lib/libc/minimal/include/sys/stat.h

```diff
@@ -1,9 +1,49 @@
 /*
- * Copyright (c) 2018 Linaro Limited
+ * Copyright (c) 2018 Intel Corporation
```

Comment:
I'd keep the Intel Copyright and just add the Linaro one.

Reply:
> I'd keep the Intel Copyright and just add the Linaro one.

Ok. Note that the original sys/stat.h was an empty file with just a comment "Dummy sys/stat.h to fulfill the requirements of certain libraries."

Reply:
Sure, just felt like we are moving most of the contents of include/posix/sys/stat.h into there.

Reply:
> I'd keep the Intel Copyright and just add the Linaro one.

D'oh, so what happened here, is that we had a dummy, empty file copyrighted Linaro. It gets overwritten by a moved file copyrighted by Intel. I didn't specifically change any copyright, just one file gets completely overwritten by another.

@galak, I hope you agree that under such circumstances, it doesn't make sense to stamp Linaro's copyright there.

Reply:
Ok, so hopefully this can be closed.

Reply:
I'm fine not having the linaro copyright.